packages/preview-middleware-client/src/adp/init.ts

```diff
@@ -8,13 +8,16 @@ import {
     showMessage,
     startPostMessageCommunication
 } from '@sap-ux-private/control-property-editor-common';
+
 import { ActionHandler } from '../cpe/types';
-import ExtensionPointService from './extension-point';
 import VersionInfo from 'sap/ui/VersionInfo';
 import { getUI5VersionValidationMessage } from './ui5-version-utils';
 
 export default async function (rta: RuntimeAuthoring) {
-    const extPointService = new ExtensionPointService(rta);
+    const { version } = (await VersionInfo.load()) as { version: string };
+    const versionParts = version.split(/[.-]/);
```

Comment:
The version might be snapshot-1.125.10

Reply:
Doing `snapshot-1.125.10.split(".")` will return `['snapshot-1', '125', '10']` and since you are interested only in the `minor` version it will work 😉

Reply:
Changed in: [3f1f9d5](https://github.com/SAP/open-ux-tools/pull/1597/commits/3f1f9d5314445754b1a015564e7a6480404ebc90)

Reply:
Great, thanks!